uber/templates/registration/discount.html

```diff
@@ -8,6 +8,9 @@
 {% csrf_token %}
 <table class="form">
 <tr>
+    <!-- Note that this is preferred first and last name.
+    May want to change, probably by sending an attendee ID to this page as a hidden field/something,
+    instead of having to re-search for them with potentially ambiguous/problematic results. -->
```

Comment:
Just making sure I'm understanding here: discount.html creates a new attendee, not modifying an existing one? I assumed modification, but from your comment I may have been wrong - if it is creating a new attendee, then I'll remove the comment here since it's not relevant.